Dockerfile

```
@@ -8,17 +8,42 @@ FROM oven/bun:1 AS frontend
+# Copy all needed files for frontend build
 COPY app/ ./app/
 COPY web/ ./web/
-
-# Strip workspaces not needed for web build, and fix trailing comma
-RUN sed -i '/"tauri"/d; /"landing"/d' package.json && \
-    sed -i -z 's/,\n  ]/\n  ]/' package.json
+COPY bun.lock CHANGELOG.md ./
+
+# Create a temporary package.json for container (with all root deps needed)
+RUN cat > package-temp.json << 'EOFPKG' && mv package-temp.json package.json
+{
```

Hadolint (2.14.0)

[error] unexpected '{'
expecting '#', '\', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, at least one space, or end of input

(DL1000)